src/priceHistory/slice.ts

```diff
@@ -0,0 +1,68 @@
+import { Draft, PayloadAction, createSlice } from '@reduxjs/toolkit'
+import BigNumber from 'bignumber.js'
+
+type PriceHistoryStatus = 'idle' | 'loading' | 'success' | 'error'
```

Comment:
'idle' isn't clear to me. What does it mean, and when does it occur? Should it be 'notSet' (assuming that's what it means)?

Reply:
seems like it is only used in the selector to indicate that there is no price history status found for the token. we could potentially use undefined/null for that, but if we do use a string for it, I'd advocate for something a bit clearer, like 'notSet' or 'notFound'

Reply:
removed in b6428a8ce1aa3b4bed5bd6948d4caf8aeb6f4f9c!